src/restapi/mm_routes.ts

```diff
@@ -0,0 +1,49 @@
+import fs from 'fs';
+
+import express, {Request, Response} from 'express';
+import {AppCall} from 'mattermost-redux/types/apps';
+
+import app from '../app/app';
+
+import {jsonStoreFileName} from '../app/constants';
```

Comment:
Can we group the imports like this?

```suggestion
import fs from 'fs';
import express, {Request, Response} from 'express';

import {AppCall} from 'mattermost-redux/types/apps';

import app from '../app/app';
import {jsonStoreFileName} from '../app/constants';
```

Reply:
yes, but lint adds an empty line between the first two.